decidim-budgets/lib/decidim/budgets/seeds.rb

```diff
@@ -13,56 +13,76 @@ def initialize(participatory_space:)
       end
 
       def call
+        component = create_component!
+
+        rand(1...3).times do
+          create_budget!(component:)
+        end
+
+        Decidim::Budgets::Budget.where(component:).each do |budget|
+          rand(2...4).times do
+            project = create_project!(budget:)
+
+            create_attachments!(attached_to: project)
+
+            Decidim::Comments::Seed.comments_for(project)
+          end
+        end
+      end
+
+      def create_component!
         landing_page_content = Decidim::Faker::Localized.localized do
           "<h2>#{::Faker::Lorem.sentence}</h2>" \
             "<p>#{::Faker::Lorem.paragraph}</p>" \
             "<p>#{::Faker::Lorem.paragraph}</p>"
         end
 
-        component = Decidim::Component.create!(
+        Decidim::Component.create!(
```

Comment:
there is no traceability method being called. 

And this snippet is present in elections, meetings, proposals
```
Decidim.traceability.perform_action!(
          "publish",
          Decidim::Component,
          admin_user,
          visibility: "all"
        ) do
          Decidim::Component.create!(params)
        end
```

Reply:
Done in [cd4694a](https://github.com/decidim/decidim/pull/12205/commits/cd4694aa8b6c452e8caf280953d668f6f9f62821)